### PR TITLE
feat(appeals): appeal schema

### PR DESCRIPTION
**Pull request by fabfab-kc**
Initial schema to track progress
Source: [consolidated spreadsheet](https://pinso365-my.sharepoint.com/:x:/r/personal/gareth_hesford_planninginspectorate_gov_uk/Documents/Documents/ODW/Appeals/HAS/Appeals FO BO output.xlsx?d=w7def75203be04b4e876771ea859df8e2&csf=1&web=1&e=9eCrNu)

### CODE REVIEW

File: schemas/appeal.schema.json [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+  "required": [],
+  "properties": {
+    "caseReference": {
+      "type": "string",
+      "examples": ["7002471"],
+      "description": "External case identifier"
+    },
+    "caseStatus": {
+      "type": "string",
+      "enum": [
```
Comment (fabfab-kc) [inline schemas/appeal.schema.json:17]: Need to add additional statuses for:
- complete
- closed
- awaiting_transfer
- transferred
- withdrawn

File: schemas/appeal.schema.json [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+    "caseValidationDate": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "description": "The date the appeal was validated in the back-office"
+    },
+    "caseValidationOutcome": {
+      "type": ["string", "null"],
+      "enum": ["valid", "invalid", "incomplete"],
+      "description": "The type of procedure for the appeal"
```
Comment (fabfab-kc) [inline schemas/appeal.schema.json:104]: Fix description: `The outcome of the validation action`

File: schemas/appeal.schema.json [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+        ]
+      },
+      "description": "A list of reasons why the appeal is incomplete",
+      "$comment": "The list will contain items only if the caseValidationOutcome is incomplete"
+    },
+    "caseExtensionDate": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "description": "When the validation outcome is incomplete, an extension may be grated to provide missing information"
```
Comment (fabfab-kc) [inline schemas/appeal.schema.json:134]: `granted`

File: schemas/appeal.schema.json [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "description": "The date the appeal decision letter is published"
+    },
+    "lpaCode": {
+      "type": "string",
+      "examples": ["7002471"],
+      "description": "A unique identifier for the Local Planning Authority",
+      "$comment": "Uses the ONS code identifying LPAs"
```
Comment (fabfab-kc) [inline schemas/appeal.schema.json:187]: Need to confirm `ONS` and example

File: schemas/appeal.schema.json [hunk 5]
```diff
@@ -0,0 +1,553 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "appeal.schema.json",
+  "title": "Appeal Document",
```
Comment (beejjacobs) [inline schemas/appeal.schema.json:4]: Appeal Case

File: schemas/appeal.schema.json [hunk 6]
```diff
@@ -0,0 +1,553 @@
...
+      "items": {
+        "type": "string",
+        "examples": ["6000456", "8004567"]
+      },
+      "description": "A list of related case references known to the appellant and the LPA"
+    },
+    "neighbouringSiteAddresses": {
+      "type": "array",
+      "items": {
+        "type": "object",
```
Comment (beejjacobs) [inline schemas/appeal.schema.json:494]: Any nested object definitions need `additionalProperties: true` as well.

**Pull request closed without merging**